File: 2026-09/src/day_11/prms_181864/Solution.java
package day_11.prms_181864;

class Solution {
    public int solution(String myString, String pat) {
        char[] arr = myString.toCharArray();
        for (int i = 0; i < arr.length; i++) {
            arr[i] = (arr[i] == 'A') ? 'B' : 'A';
        }

        return new String(arr).contains(pat) ? 1 : 0;
    }
}

File: 2026-09/src/day_11/prms_181864/Solution.cpp
#include <bits/stdc++.h>
using namespace std;

int solution(string myString, string pat) {
    for (char& c : myString) {
        c = (c == 'A') ? 'B' : 'A';
    }

    return myString.find(pat) != -1;
}
